src/instructions/proctored_exam/ErrorProctoredExamInstructions.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Container, Hyperlink } from '@edx/paragon';
+import ExamStateContext from '../../context';
+import Footer from './Footer';
+
+const ErrorProctoredExamInstructions = () => {
+  const state = useContext(ExamStateContext);
+  const { link_urls: linkUrls, platform_name: platformName } = state.proctoringSettings;
+  const contactUsUrl = linkUrls && linkUrls.contact_us;
+
+  return (
+    <div>
+      <Container className="border py-5 mb-4">
+        <h3 className="h3">
+          <FormattedMessage
+            id="exam.ErrorProctoredExamInstructions.title"
+            defaultMessage="Error with proctored exam"
+          />
+        </h3>
```

Comment:
I think we use div element for title everywhere in the app, so maybe change h3 to div so we are consistent with it

Reply:
Changed